### PR TITLE
feat: add health_check_timeout option

### PR DESCRIPTION
**Pull request by rkul**
## what

Add health check timeout option

## why

Impossible to set health check interval to less than 10 seconds, because it should be less than health check timeout

## references

https://docs.aws.amazon.com/elasticloadbalancing/latest/application/target-group-health-checks.html

image

### CODE REVIEW

**Review by soar — APPROVED** [review]

File: variables.tf [hunk 1]
```diff
@@ -264,6 +264,12 @@ variable "health_check_interval" {
   description = "The duration in seconds in between health checks"
 }
 
+variable "health_check_timeout" {
+  type        = number
+  default     = 10
```
Comment (hans-d) [inline variables.tf:269]: Prevent providing a default that is not matching the default from aws. Better to have the default being null, keeping it backward compatible.

**Review by hans-d — CHANGES_REQUESTED** [review]

File: variables.tf (hunk 1 above)
Comment (rkul) [reply variables.tf]: @hans-d yeah, makes sense, fixed

**Review by soar — APPROVED** [review]

**Review by hans-d — APPROVED** [review]

**Pull request merged**